Disaster recovery without network restore (#486)
NethServer/docs: administrator-manual/en/disaster_recovery.rst

connectivity and login again on the Server Manager.

.. _skip-network-restore-section:

Skip network restore
--------------------

Network configuration is restored by default, but sometimes it is necessary to restore an 
installation on a different hardware without migrating the network configuration.
This is a common scenario when moving a virtual machine from a VPS provider to another.

To disable the network restore, make sure to disable the :guilabel:`Restore network configuration` option from
the new Server Manager.

Since some application configurations depend on network interface names, not everything can be automatically restored.

DHCP
^^^^

DHCP servers on non-existing interfaces will be deleted.
If needed, please reconfigure the DHCP from the Server Manager.
See also :ref:`dhcp-section` for more general information.

Samba Active Directory
^^^^^^^^^^^^^^^^^^^^^^

Samba Active Directory requires a network bridge in the green zone for the local running container.
If the bridge already exists, the container will continue running after the restore.

If the bridge does not exist anymore, Samba Active Directory is stopped.
To enable it again:

> 

Author: ```suggestion
- make sure the old IP address of the container is still inside the new green network. If not, set it from command line, eg: ::
    config setprop nsdc bridge IpAddress 192.168.1.2
```

Reviewer: I want to make some tests: I'm not sure the nethserver-dc-save event is good. We may use nethserver-dc-change-ip instead.

connectivity and login again on the Server Manager.

.. _skip-network-restore-section:

Skip network restore
--------------------

> Network configuration is restored by default, but sometimes it's necessary to restore an 

Reviewer: Do not use contracted forms

```suggestion
Network configuration is restored by default, but sometimes it is necessary to restore an 
```

installation on a different hardware without migrating the network configuration.
This is a common scenario when moving a virtual machine from a VPS provider to another.

To disable the network restore, make sure to disable the :guilabel:`Restore network configuration` option from
the new Server Manager.

Since some application configurations depend on network interface names, not everything can be automatically restored.

DHCP
^^^^

> DHCP servers on non-existing interface will be deleted.

Reviewer: ```suggestion
DHCP servers on non-existing interfaces will be deleted.
```

If needed, please reconfigure the DHCP from the Server Manager.
See also :ref:`dhcp-section` for more general information.

Samba Active Directory
^^^^^^^^^^^^^^^^^^^^^^

> Samba Active Directory requires a network bridge for the local running container.

Reviewer: ```suggestion
Samba Active Directory requires a network bridge in the green zone for the local running container.
```

If the bridge already exists, the container will continue running after the restore.

> If the bridge doesn't exists anymore, Samba Active Directory is stopped.

Reviewer: ```suggestion
If the bridge does not exist anymore, Samba Active Directory is stopped.
```

Web proxy
^^^^^^^^^

Web proxy priority rules using non-existing zones will be disabled.
Before re-enabling such rules, make sure the zones have been created.

More info on priority rules: :ref:`squid_rules-section`.

OpenVPN tunnels
^^^^^^^^^^^^^^^

OpenVPN tunnel servers contain a field named :guilabel:`Public address`.
If such field uses only public DNS names, no action is required.
Otherwise, insert the new public IP address inside the field and update tunnel clients accordingly.

See also OpenVPN :ref:`ovpn_tunnel-section`.

OpenVPN roadwarrior
^^^^^^^^^^^^^^^^^^^

OpenVPN roadwarrior server exposes a field named :guilabel:`Contact this server on public IP / host`.
If such field uses only public DNS names, no action is required.
Otherwise, insert the new public IP address inside the field and update roadwarrior clients accordingly.

See also OpenVPN :ref:`ovpn_roadwarrior-section`.

IPSec tunnels
^^^^^^^^^^^^^

Only IPSec tunnels configured with a dynamic red interface will be disabled.
Access the Server Manager, edit the disabled tunnel by selecting a new red interface and enable it again.

More info at :ref:`ipsec-section`.

Dedalo hotspot
^^^^^^^^^^^^^^

> Dedalo hotspot will be disabled if the system doesn't have a network interface configured with the hotspot role.

Reviewer: ```suggestion
Dedalo hotspot will be disabled if the system does not have a network interface configured with the ``hotspot`` role.
```

If the Dedalo is disabled, just reconfigure following :ref:`dedalo-section` chapter.

ntopng
^^^^^^

> ntopng must be reconfigured. Access the :guilabel:`Bandwidth monitor` page inside old Server Manager.

Reviewer: ```suggestion
ntopng must be reconfigured. Access the :guilabel:`Bandwidth monitor` page inside the old Server Manager.
```
